SUPPORT.md

```diff
@@ -0,0 +1,30 @@
+# Support
+
+This article explains where to get help with Peribolos.
+Please read through the following guidelines.
+
+> 👉 **Note**: before participating in our community, please read our
+> [code of conduct](CODE_OF_CONDUCT.md).
+> By interacting with this repository, organization, or community you agree to
+> abide by its terms.
+
+## Asking quality questions
+
+Questions, bug reports and feature requests can go to [GitHub issues](https://github.com/open-services-group/peribolos-as-a-service/issues/new).
+
+Help us help you! Spend time framing questions and add links and resources. Spending the extra time up front can help save everyone time in the long run. Here are some tips:
+
+* [Talk to a duck](https://rubberduckdebugging.com)!
+* Don’t fall for the [XY problem](https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem/66378#66378)
+* Search to find out if a similar question has been asked
+* Try to define what you need help with:
+  * Is there something in particular you want to do?
+  * What problem are you encountering and what steps have you taken to try and fix it?
+  * Is there a concept you don’t understand?
+* Provide sample code or a video, if possible
+* Screenshots can help, but if there’s important text such as code or error messages in them, please also provide those as text
+* The more time you put into asking your question, the better we can help you
+
+## Contributions
```

Comment:
how about we move this to the readme page? it would be easy for contributors in that way.

Reply:
Good call. What if we leave it here and add a similar note to README, wdyt @harshad16 ? I think it makes sense to have it it both places. :slightly_smiling_face:

Reply:
that would be great as well :)